src/LibraryManager.Vsix/UI/HideResizeButtons.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Windows;
+
+namespace Microsoft.Web.LibraryManager.Vsix.UI
+{
+    // Extension class for <seealso cref="System.Windows.Window"> to hide minimize and maximize buttons.
+    internal static class HideResizeButtons
```

Comment:
No. Setting window style to ToolWindow(instead of none) hides the resize buttons by default but it's themed. Since we want a default non themed window, I had to do this explicitly. So this hides the resize buttons and I added gripper for resizing.